asserter/network.go

```diff
@@ -78,6 +78,27 @@ func Version(version *types.Version) error {
 	return nil
 }
 
+// SyncStatus ensures any types.SyncStatus is valid.
+func SyncStatus(status *types.SyncStatus) error {
```

Comment:
Since this always returns error (or nil), I'd prefer a name like `SyncStatusError` or `ValidateSyncStatus`. What do you think?

Reply:
This is consistent with all other functions in `asserter`. Currently, the pattern is that the name of the `asserter` method is the name of the struct it is asserting.

If we want to migrate to your naming strategy, I'd recommend we do that in a separate PR.